.circleci/config.yml

```diff
@@ -0,0 +1,202 @@
+          requires:
+            - build
+          filters:
+            branches:
+              only:
+                - develop
+
+  api-triggered-delivery: # Workflow for manual delivery
+    when: << pipeline.parameters.api_triggered >>
+    jobs:
+      - build:
+          production: << pipeline.parameters.production >>
+      - bump_version:
+          shouldBumpVersion: << pipeline.parameters.production >>
+          requires:
+            - build
+      - deliver:
+          production: << pipeline.parameters.production >>
+          requires:
+            - build
+            - bump_version
+
+  bi-weekly-production-delivery:
+    triggers:
+      - schedule:
+          cron: "0 5 1,15 * *" # For sanity checks for this syntax: https://crontab.guru/
+          filters:
+            branches:
+              only:
+                - develop
+    jobs:
+      - build:
+          production: true
+      - bump_version:
+          requires:
+            - build
+      - deliver:
+          type: approval
+          production: true
+          requires:
+            - build
+            - bump_version
+
+jobs:
+  build:
+    parameters:
+      production:
+        description: Whether to build for production.
+        type: boolean
+    docker:
+      - image: circleci/node:12.16.3
+    resource_class: small
+    environment:
+      TOTAL_CPUS: 1 # For resource_class small
+      TZ: "Europe/Berlin"
+    shell: /bin/bash -eo pipefail
+    steps:
+      - run:
+          name: "Show current CPU count"
+          command: |
+            echo "CPU count is ${TOTAL_CPUS}"
+      - add_ssh_keys: # Install
+          fingerprints:
+            - "30:f4:98:9f:1c:72:d1:c0:17:9f:57:05:74:a6:44:50"
+      - checkout
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          keys:
+            - 2-yarn-{{ arch }}-{{ checksum "yarn.lock" }}
+            - 2-yarn-{{ arch }}-
+      - run:
+          name: Install Dependencies
+          command: yarn install --frozen-lockfile
+      - save_cache:
+          name: Save Yarn Package Cache
+          key: 2-yarn-{{ arch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.cache/yarn
+            - node_modules
+      - run:
+          name: Debug version bump
+          command: tools/bump-version bump-version --dry-run
+      - run:
+          name: Bump version
+          command: tools/bump-version bump-version --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --branch ${CIRCLE_BRANCH}
+      - run:
+          name: Check linting
+          command: yarn run lint
+      - run:
+          name: Check flow
+          command: yarn run flow:check-now --max-workers $TOTAL_CPUS
+      - run:
+          name: Unit tests
+          command: yarn run test --ci --maxWorkers $TOTAL_CPUS
+      - run:
+          name: Test coverage
+          command: yarn run test:coverage --ci --maxWorkers $TOTAL_CPUS
```

Comment:
Minor: A comment that we do not do anything with the results. We should run it nonetheless as it can throw errors which are not thrown in the previous step.